samples/qa-a11y/src/views/Alert.js

```diff
@@ -6,8 +6,13 @@ import useArrayState from '../components/useArrayState';
 
 import appCss from '../App/App.module.less';
 
+
+
 const AlertView = () => {
 	const [open, handleOpen] = useArrayState(8);
+	const svgGenerator = (width, height, bgColor, textColor, customText) => (
+		`data:image/svg+xml,%3Csvg xmlns='http://www.w3.org/2000/svg' viewBox='0 0 ${width} ${height}' class='img-fluid rounded mx-auto d-block' width='${width}' height='${height}'%3E%3Crect width='${width}' height='${height}' fill='%23${bgColor}'%3E%3C/rect%3E%3Ctext x='50%25' y='50%25' dominant-baseline='middle' text-anchor='middle' font-size='36px' fill='%23${textColor}'%3E${customText}%3C/text%3E%3C/svg%3E`
+	);
```

Comment:
I think class attribute need to be removed from our code.
```suggestion
	const svgGenerator = (width, height, bgColor, textColor, customText) => (
		`data:image/svg+xml,%3Csvg xmlns='http://www.w3.org/2000/svg' viewBox='0 0 ${width} ${height}' width='${width}' height='${height}'%3E` +
		`%3Crect width='${width}' height='${height}' fill='%23${bgColor}'%3E%3C/rect%3E` +
		`%3Ctext x='50%25' y='50%25' dominant-baseline='middle' text-anchor='middle' font-size='36px' fill='%23${textColor}'%3E${customText}%3C/text%3E%3C/svg%3E`
	);
```